content/docs/studio/user-guide/connect-custom-gitlab-server.md

```diff
@@ -0,0 +1,44 @@
+# Custom GitLab Server Connection
+
+If your team’s Git repositories are on a self-hosted GitLab server, you can set
+up a connection to this server such that all your team members can connect to
+the Git repositories on this server. Refer to the
+[GitLab docs](https://about.gitlab.com/install/) for more details about
+self-hosted GitLab servers.
+
+To connect to your self-hosted GitLab server, you will need the following:
+
+- A URL for your GitLab server. This URL should be accessible to Studio. That
+  is, it could be a publicly accessible URL. Or, if Studio is deployed
```

Comment:
Changed it to `This URL should either be publicly accessible, or, if Studio is deployed on-premises in a private network, then it could also be a URL in the same private network so that Studio can access it.`

Reply:
How about

> A publicly accessible URL for your GitLab server, or a URL in the same private network if Studio is deployed on-premises.

Reply:
Changed to `A URL for your GitLab server. The URL should either be publicly accessible, or in the same private network as Studio if Studio is deployed on-premises.`